src/admin/settings/settings.html

```diff
@@ -0,0 +1,126 @@
+<div ng-if="!platformSettings.readOnly" class="form-group settings-box-medium">
+    <div class="settings-box-content">
+        <legend class="frame-title settings-box-header">{{'admin.settings.upload' | translate}}</legend>
+        <div class="settings-form">
+            <legend>{{'admin.platform.settings' | translate}}</legend>
+            <form class="form-inline inside" id="settingsFileForm" action="module/admin/api/settings/platform/upload" method="post" enctype="multipart/form-data">
+                <fieldset>
+                    <label class="control-label input-label">{{'admin.settings.fileChosen' | translate}}</label>
+                    <div class="btn-group">
+                        <span class="btn btn-default">
+                            <span>{{'server.button.selectFile' | translate}}</span>
+                            <input type="file" id="fileInput" name="file"/>
+                        </span>
+                    </div>
+                    <span class="btn btn-primary" ng-click="uploadSettings()">{{'admin.submit' | translate}}</span>
+                </fieldset>
+            </form>
+        </div>
+    </div>
+</div>
+<div ng-if="!platformSettings.readOnly" class="form-group settings-box-small">
+    <div class="settings-box-content">
+        <legend class="frame-title settings-box-header">{{'admin.settings.location' | translate}}</legend>
+        <form class="form-inline inside" id="settingsLocationForm" method="post" ng-submit="uploadFileLocation()">
+            <fieldset class="form-content">
+                <label class="control-label input-label">{{'admin.settings.location.new' | translate}}</label>
+                <input class="form-control" type="text" id="settingsLocation" name="settingsLocation" ng-model="location"/>
+                <input type="submit" value="{{'admin.submit' | translate}}" class="btn btn-primary"/>
+            </fieldset>
+        </form>
+    </div>
+</div>
+<div class="form-group settings-box-small">
+    <div class="settings-box-content">
+        <legend class="frame-title settings-box-header">{{'admin.settings.importexport.header' | translate}}</legend>
+        <form class="form-inline inside">
+            <fieldset class="form-content">
+                <label class="control-label input-label">{{'admin.settings.importexport.export' | translate}}</label>
+                <input type="button" ng-click="exportConfig()" value="{{'admin.settings.importexport.exportButton' | translate}}" class="btn btn-default"/>
+            </fieldset>
+        </form>
+    </div>
+</div>
+<div class="clearfix"></div>
+<div class="form-group">
+    <div class="settings-box-content" ng-show="platformSettings.settingsList.length > 0">
```

Comment:
I don't think we need this ng-show element -- I'd remove the div and just leave the ng-repeat...